### DISCUSSION TITLE
Is ist possible to get QField to access attachment from WebDAV Storage?

### DISCUSSION THREAD

**Discussion by soester** [opengisch/QField#2707] [Q&A]
We have a large collection of photos from previous field work. It would be great if we could access those photos in the field, without copying them to the tablets.

If configuring the Widget Type 'Attachment' with the 'WebDAV Storage' option in QGIS, it is possible to display attachments from WebDAV storage in the Attribute Form (eg. pictures).
But when creating an offline project, QField is unable to display these attachments. I think this is because the Authentication-Database is not available in QField. I didn't manage to store the credentials in the Qgis Project File itself (Don't know if this is an Issue on QGIS side).

Should the 'WebDAV Storage' option for attachments work in QField? Or is this not implemented?

**Comment by suricactus** [comment]
No, it is not currently possible. It is also not planned for the near future.

What is planned for the future is having the attachments as push only. In addition, they can be configured to be directly served from the QFieldCloud server on demand, if there is internet connection. Thus it saves packaging time and data transfer when downloading your project, removing the need to download images that you might not need.

Another alternative is to use `qfieldcloud-cli` to download and sync your images in a way you prefer.

 If you are interested in any of the (not) planned solution being implemented, please drop us an email on info [at] opengis.ch.
